### PR TITLE
docs: translate hertz/reference/version & json

### PR DESCRIPTION
**Pull request by stephenzhang0713**
docs: translate hertz/reference/version & json

### CODE REVIEW

**Comment by stephenzhang0713** [comment]
@GuangmingLuo Would you like to review this translation for me?

**Comment by GuangmingLuo** [comment]
ref: https://github.com/cloudwego/hertz/issues/86

File: content/en/docs/hertz/reference/json.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "JSON Marshal Library"
+linkTitle: "JSON Marshal Library"
+weight: 3
+description: >
+
+---
+
+
+Hertz integrates [sonic](https://github.com/bytedance/sonic) by default as a json marshal library for serializing data with the ctx.JSON interface. sonic is an ultra-high performance golang json library, see Sonic [README](https://github.com/bytedance/sonic) for details.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/reference/json.md:10]: 1. add code block for `ctx.JSON`
2. sonic -> Sonic 
3. please fix those in zh doc as well

File: content/en/docs/hertz/reference/json.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The following are currently required to enable Sonic:
+- Go 1.15/1.16/1.17/1.18
+- Linux / darwin OS / Windows
+- Amd64 CPU with AVX instruction set
+
+Automatically fallback to golang's encoding/json library if not supported.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/reference/json.md:17]: ... if the above requirements are not satisfied

File: content/en/docs/hertz/reference/json.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+In addition,sonic
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/reference/json.md:19]: In addition, Sonic (space and upper case)

File: content/en/docs/hertz/reference/json.md (hunk 1 above)
Comment (stephenzhang0713) [reply content/en/docs/hertz/reference/json.md]: done

File: content/en/docs/hertz/reference/json.md (hunk 2 above)
Comment (stephenzhang0713) [reply content/en/docs/hertz/reference/json.md]: done

File: content/en/docs/hertz/reference/json.md (hunk 3 above)
Comment (stephenzhang0713) [reply content/en/docs/hertz/reference/json.md]: done

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**